### PR TITLE
Fixes #87

### LINKED ISSUE #87: empty group file gives misleading error
When loading a zero length group file (by mistake) I got:
```
drand: error loading group argument: time: invalid duration 
```

### PR DESCRIPTION
**Pull request by PizzaWhisperer**
- Add test to reject empty group file as argument.
Enhancement: change error message when unsuccessfully loading a groupfile to give more info about what went wrong (corrupted groupfile, wrong path, ..)

### CODE REVIEW

File: daemon.go [hunk 1]
```diff
@@ -52,3 +52,15 @@ func stopDaemon(c *cli.Context) error {
...
+
+func stopCmd(c *cli.Context) error {
+	conf := contextToConfig(c)
+	fs := key.NewFileStore(conf.ConfigFolder())
+	var drand *core.Drand
+	drand, err := core.LoadDrand(fs, conf)
+	if err != nil {
+		slog.Fatal(err)
+	}
+	drand.Stop()
```
Comment (nikkolasg) [inline daemon.go]: Remember we're in client / daemon mode now, so you can't just load drand and then stop it, since it's already running. You have to issue a gRPC command to the daemon so that it stops itself. And if you do that, please do it in a separate PR so we have a better visibility :)

**Comment by nikkolasg** [comment]
I'm fine with merging if you remove the part about the stop command :) LGTM
